docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
```

Comment:
Every plugin that implements an rsync distributor will now have to depend on the rsync package. I'll adjust the spec file for pulp_rpm to reflect this.